src/index.js

```diff
@@ -27,36 +27,86 @@ const fetchData = () => {
       tripsData: dataSet[1].trips,
       destinationsData: dataSet[2].destinations, 
     }).then(dataSet => {
+      const loginButton = document.getElementById('login-button')
+      loginButton.addEventListener('click', domUpdates.submitLogin);
       allTravelers = new AllTravelers(dataSet.travelersData, dataSet.tripsData, dataSet.destinationsData); 
       traveler = new Traveler(2, dataSet.travelersData, dataSet.tripsData, dataSet.destinationsData);
-      domUpdates.displayWelcome(traveler);
-      domUpdates.displayTravelersTrips(traveler);
-      tripDestinations.addEventListener('click', domUpdates.displayDestinationList(allTravelers));     
-      submitBtn.addEventListener('click', function() {
-        // domUpdates.displayNewTripCost(traveler);
-        postTrip(allTravelers, traveler);
-      });
-      domUpdates.displayTravelerCosts(traveler);
+      travelerPage(traveler, allTravelers);
+      agentPage(allTravelers);
     })
     .catch(error => console.log(error.message));
 }
 
-const loginButton = document.getElementById('login-button')
-const tripDestinations= document.getElementById('trip-destinations')
-const submitBtn = document.getElementById('submit-btn');
+// function getId(data) {
+//     const username = document.getElementById('username').value;
+//     const travelerID = Number(username.split('traveler')[1])
+//   }
 
-loginButton.addEventListener('click', domUpdates.submitLogin); 
+function travelerPage(traveler, allTravelers) {
+  const tripDestinations = document.getElementById('trip-destinations');
+  const submitBtn = document.getElementById('submit-btn');
+  const costBtn = document.getElementById('cost-btn');
+  const travelerDisplay = document.querySelector('.traveler');
+  const estimatedCost = document.querySelector('.estimated-cost')
+  if (travelerDisplay) {
+    domUpdates.displayWelcome(traveler);
+    domUpdates.displayTrips('.trip-info', traveler, "", 'getTripFormat');
+    tripDestinations.addEventListener('click', domUpdates.displayDestinationList(allTravelers));     
+    submitBtn.addEventListener('click', function() {
+      postTrip(allTravelers, traveler);
+    });
+    costBtn.addEventListener('click', function() {
+      domUpdates.displayNewTripCost(traveler);
+      document.addEventListener('click', function(event) {
+        const eventParent = event.target.parentNode.parentNode.parentNode.parentNode.parentNode.classList.contains('overlay')
```

Comment:
This is a lot of parentNodes!

Reply:
yeah....I don't want to talk about how long I spent trying to not go that route :(